src/components/accessories/feedback/utils.ts

```diff
@@ -0,0 +1,13 @@
+import * as params from "../../../customization/feedback.json";
+import { IFeedbackProps } from "./type";
+
+const collectorUrl =
+  "https://openhospital.atlassian.net/s/d41d8cd98f00b204e9800998ecf8427e-T/r5gghz/b/3/bc54840da492f9ca037209037ef0522a/_/download/batch/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector.js?locale=en-US&collectorId=53f7755c";
+
+export const feedbackProps: IFeedbackProps = {
```

Comment:
The name of this constant could be misleading because of the "...Props" part (it's not used as a React `props` object). I'd suggest you rename it as `feedbackConfig` or something like that.

Reply:
@almeida1492 I deed it, please take a look at it.